mmv1/third_party/terraform/resources/resource_bigtable_instance.go

```diff
@@ -82,6 +78,13 @@ func resourceBigtableInstance() *schema.Resource {
 							ValidateFunc: validation.StringInSlice([]string{"SSD", "HDD"}, false),
 							Description:  `The storage type to use. One of "SSD" or "HDD". Defaults to "SSD".`,
 						},
+						"kms_key_name": {
```

Comment:
Note for context:
This field is optional+computed and deletion protection is available. Introducing this as a new ForceNew field should have lower risk than before.